assimilation_code/programs/readme.rst

```diff
@@ -0,0 +1,195 @@
+   especially a new one.
+
+`perturb_single_instance/perturb_single_instance <perturb_single_instance/perturb_single_instance.html>`_
+   Generate an ensemble of perturbed ensemble member restart files.
+   (Alternatively, you might perturb the model state using ``model_nml`` variables).
+ 
+`gen_sampling_err_table/gen_sampling_err_table <gen_sampling_err_table/gen_sampling_err_table.html>`_
+   Computes a table of values needed to apply Sampling Error Correction (SEC),
+   which corrects covariances based on small sample size statistics.
+ 
+Creating Observation Sequence Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+obs_utils/create_obs_grid
+   Create a set of observations located on a regular grid.  Obs have no data values.
+
+`create_fixed_network_seq/create_fixed_network_seq <create_fixed_network_seq/create_fixed_network_seq.html>`_ 
+   Reads observation sequence file information from standard input 
+   and replicates it multiple times in a second observation sequence file, at user specified dates. 
+ 
+obs_utils/obs_timejitter
+   Randomly perturb the times of the observations in a (usually) set_def.out file.
+   Writes the results to (usually) obs_seq.in.
+
+`create_obs_sequence/create_obs_sequence <create_obs_sequence/create_obs_sequence.html>`_
+   Creates a short andor synthetic observation sequence file using values read from standard input.
+ 
+Querying Observation Sequence Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+obs_utils/obs_info
+   Summarize obs types, times, counts found in observation sequence file(s).
+
+obs_utils/obs_assim_count
+   Prints out a quick table of obs types and counts, overall start and stop times, 
+   and metadata strings and counts.  See obs_diag for more.
+   There is an older version in the obs_assim_count directory.
+
+`obs_seq_coverage/obs_seq_coverage <obs_seq_coverage/obs_seq_coverage.html>`_
+   Queries a set of observation sequence files to determine which observation locations report
+   frequently enough to be useful for a verification study.
+ 
+obs_total_error/obs_total_error
+   Prints the total error in the mean and spread from an obs_seq file 
+   which has been through both perfect_model_obs and filter, so it has copies
+   'truth', 'ensemble mean', and 'ensemble spread'.
+   You can get more information by running the obs_diag program.
+
+Changing Observation Sequence Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`obs_keep_a_few/obs_keep_a_few <obs_keep_a_few/obs_keep_a_few.html>`_
+   Creates an output observation sequence file that is shorter than the input obs_seq file.
+ 
+`obs_selection/obs_selection <obs_selection/obs_selection.html>`_
+   Extracts observations out of one or more obs_sequence files
+   according to a  list of observation types, times, and locations.
+   The list is usually created by :doc:`obs_seq_coverage/obs_seq_coverage`, 
+   but can be an observation sequence file,
+ 
+`obs_sequence_tool/obs_sequence_tool <obs_sequence_tool/obs_sequence_tool.html>`_
```

Comment:
since you said at the start of this file that the order inside the sections reflects the expected utility/use of the programs, i'd list this one first.  i'd move keep_a_few after obs_common_subset, and i'd put obs_selection, obs_coverage and obs_seq_verify together because they're sort of a matched set of executables.

Reply:
incorporated all most all of that.